ovos_core/intent_services/converse_service.py

```diff
@@ -54,7 +54,12 @@ def get_active_skills(self, message=None):
             active_skills (list): ordered list of skill_ids
         """
         session = SessionManager.get(message)
-        return [skill[0] for skill in session.active_skills]
```

Comment:
this can be simplified, if the goal is to deduplicate you can return `list(set([skill[0] for skill in session.active_skills]))`

Reply:
actually, if there are duplicates to start with that's another bug, activating a skill should update timestamp, not add a second entry

Reply:
The problem here that `set` is not ordered, and we want to keep the activation timestamp prio

Reply:
checked again and i must have dreamed the additional entries. Will revert